src/main/java/gr/cytech/chatreminderbot/rest/BotResource.java

```diff
@@ -14,11 +14,12 @@
 
 @Path("/services")
 public class BotResource {
-    private final static Logger logger = LoggerFactory.getLogger(BotResource.class.getName());
-    private String space_id;
+    private static final Logger logger = LoggerFactory
+            .getLogger(BotResource.class.getName());
```

Comment:
why not `xxx.class` instead of `xxx.class.getName()` ? This goes for all logger declarations in all classes.
Also, this will most probably keep you within the 120-chars per line limit